jaxrs-tck/src/main/java/jakarta/ws/rs/tck/uribuilder/UriBuilderIT.java

```diff
@@ -61,18 +61,38 @@ public final void shouldBuildValidInstanceFromScratch()
     }
 
     /**
-     * Verifies that no invalid URI can be created from scratch.
+     * Verifies that {@code UriBuilder#build()} throws a {@link UriBuilderException}
+     * when it would be asked to create an invalid URI.
      * 
      * @throws ExecutionException   if the instance didn't boot correctly
      * @throws InterruptedException if the test took much longer than usually
      *                              expected
      */
     @Test
-    public final void shouldNotBuildInvalidUriFromScratch() throws InterruptedException, ExecutionException {
+    public final void shouldThrowUriBuilderExceptionOnSchemeOnlyUri() throws InterruptedException, ExecutionException {
         // given
-        final UriBuilder uriBuilder = UriBuilder.newInstance();
```

Comment:
Why do you want to allow `UriBuilder.newInstance();` to be called? I'd rather vote for having an *additional* test that disallows scheme-only uri builders instead of allowing completely invalid builders.

Reply:
Sorry, I don't understand your comment.  `UriBuilder.newInstance()` was there from before - your change, I believe? The reason I am removing that is that `UriBuilder.newInstance().build()` is valid as it is equivalent to `URI.create("")` which is allowed.  Current implementations are failing on the current line 76, and as this test is not part of a new feature that is requiring new restrictions on the API, we should not be adding it. However, since a scheme-only URI is not allowed by the `javax.net` APIs and by existing JAX-RS implementation, it is a valid test.

Reply:
`newInstance()` was added in https://github.com/eclipse-ee4j/jaxrs-api/pull/641, and unfortunately we did not really discuss at that point, what the actual outcome of `UriBuilder.newInstance()` shall actuall be like - is it really the same as `URI.create("")`, or is it forbidden? So when I wrote the UriBuilderIT, I added that test that explicitly forbids it -- and that test was accepted by the committers, hence it is authoritative until decided by the committers.

In fact I do not have any problem if we change the test to actually proof the exact oppostite, but then please add your test as an *additional* test and *change* my existing test to proof that the outcome really is `URI.create("")` -- but do not leave the case `newInstance();` completely **untested** (hence: undecided by the TCK as an authoritative means).

Having said that, I will remove my veto once either you keep my test (and I open a PR turning it into the opposite) *or* you simply turn my test in the opposite in *this* PR. :-)

Reply:
Changes are in.